charactersheet/charactersheet/models/common/armor.js

```diff
@@ -25,7 +25,12 @@ function Armor() {
     self.armorStealthOptions = ko.observableArray(Fixtures.armor.armorStealthOptions);
 
     self.acLabel = ko.pureComputed(function() {
-        return 'AC ' + self.armorClass();
+        if (self.armorClass().replace(/^\s+|\s+$/g, '')) {
```

Comment:
Yeah it's the most browser friendly way of stripping whitespace I could find.  If there is whitespace before or after AC, the expected results don't occur.